bdb/rep.c

```diff
@@ -5508,11 +5506,6 @@ void *watcher_thread(void *arg)
 
         BDB_READLOCK("watcher_thread");
 
-        if (!gbl_new_snapisol_asof) {
-            DB_LSN lsn = {0};
-            bdb_clean_pglogs_queues(bdb_state, lsn, 0);
```

Comment:
this function does anything if `gbl_new_snapisol` is enabled, so deleted that function and all of its invocations.